apps/components-e2e/src/components/confirmation-dialog/confirmation-dialog.e2e.ts

```diff
@@ -90,3 +91,17 @@ test('should should not open dialog when backdrop is enabled', async (testContro
     .expect(successDialog.count)
     .eql(0);
 });
+
+test('should propagate attribute to overlay', async (testController: TestController) => {
+  await testController
+    .expect(saveButton.count)
+    .eql(0)
+    .expect(dirtyDialog.count)
+    .eql(0)
+    .expect(successDialog.count)
+    .eql(0)
+    .click(openDialogButton)
+    .wait(40000)
```

Comment:
this is definitive to high! please remove the wait. and reduce the speed at the click before to 0.3